sklearn/pipeline.py

```diff
@@ -1615,7 +1642,20 @@ def fit_transform(self, X, y=None, **fit_params):
             The `hstack` of results of transformers. `sum_n_components` is the
             sum of `n_components` (output dimension) over transformers.
         """
-        results = self._parallel_func(X, y, fit_params, _fit_transform_one)
+        if _routing_enabled():
+            routed_fit_transform_params = process_routing(
```

Comment:
```suggestion
            routed_params = process_routing(
```

throughout the codebase, we're been consistently useing `routed_params` everywhere, it's shorter and more consistent with the rest of the codebase.